packages/web-runtime/src/pages/account.vue

```diff
@@ -24,6 +24,13 @@
             </oc-button>
           </div>
         </template>
+        <oc-table-tr class="account-page-info-profile-picture">
+          <oc-table-td>{{ $gettext('Profile picture') }}</oc-table-td>
+          <oc-table-td>{{ $gettext('Max. 10MB, JPG, PNG') }}</oc-table-td>
```

Comment:
Thank you for removing the `10` from the translated strings in the AvatarUpload, but now you've introduced it here instead 😅